lib/index.ts

```diff
@@ -20,11 +20,7 @@ export interface JwtPayload {
   jti?: string;
 }
 
-export class InvalidTokenError extends Error {
-  constructor(message: string) {
```

Comment:
Since this constructor only passes the same arguments as the class it extends, I've removed it so that it will fall back to the parent constructor.